rewrite-java/src/main/java/org/openrewrite/java/ReplaceUUIDRandomId.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2020 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java;
```

Comment:
I think we have a `recipes` subpackage which we use for recipes transforming recipes: https://github.com/openrewrite/rewrite/tree/main/rewrite-java/src/main/java/org/openrewrite/java/recipes

Since the implementation depends on rewrite-java, I think that would be a good place.

Reply:
Ah, I didn't know we had a `recipes` sub package for transformations, that's perfect.